factory/cryptoFactory.go

```diff
@@ -0,0 +1,57 @@
+package factory
+
+import (
+	"github.com/multiversx/mx-chain-core-go/core/pubkeyConverter"
+	crypto "github.com/multiversx/mx-chain-crypto-go"
+	"github.com/multiversx/mx-chain-crypto-go/signing"
+	"github.com/multiversx/mx-chain-crypto-go/signing/ed25519"
+	logger "github.com/multiversx/mx-chain-logger-go"
+	"github.com/multiversx/mx-multi-factor-auth-go-service/core"
+	"github.com/multiversx/mx-sdk-go/blockchain/cryptoProvider"
+	"github.com/multiversx/mx-sdk-go/builders"
+)
+
+const (
+	userAddressLength = 32
+)
+
+var log = logger.GetOrCreate("factory")
+
+// cryptoComponentsHolder will hold core crypto components
+type cryptoComponentsHolder struct {
```

Comment:
this is not a general crypto components holder, but rather a wallet crypto components holder. maybe we can inject from the configuration the key type (wallet, bls) instead of 'hardcoding' them to wallet ones (bech32)

Reply:
added pubkey configuration to config